libs/com.etendorx.generate_entities/src/main/java/com/etendorx/gen/generation/GenerateEntities.java

```diff
@@ -0,0 +1,282 @@
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.io.Writer;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.etendorx.base.session.OBPropertiesProvider;
+import org.openbravo.base.model.Entity;
+import org.openbravo.base.model.ModelProvider;
+
+import com.etendoerp.etendorx.model.ETRXModelProvider;
+import com.etendorx.gen.beans.Metadata;
+import com.etendorx.gen.beans.Projection;
+import com.etendorx.gen.beans.Repository;
+import com.etendorx.gen.commandline.CommandLineProcess;
+import com.etendorx.gen.generation.interfaces.EntityGenerator;
+import com.etendorx.gen.metadata.MetadataContainer;
+import com.etendorx.gen.process.GenerateProtoFile;
+import com.etendorx.gen.util.TemplateUtil;
+
+public class GenerateEntities {
+  public static final String ERROR_GENERATING_FILE = "Error generating file: ";
+  public static final String GENERATING_FILE = "Generating file: ";
+  public static final String MODULES_GEN = "modules_gen";
+  public final static String GENERATED_DIR = "/../build/tmp/generated";
+  private static final Logger log = LogManager.getLogger();
+  private String basePath;
+  private String propertiesFile;
+
+  public String getBasePath() {
+    return basePath;
+  }
+
+  public void setBasePath(String basePath) {
+    this.basePath = basePath;
+  }
+
+  public void setFriendlyWarnings(boolean doFriendlyWarnings) {
+    OBPropertiesProvider.setFriendlyWarnings(doFriendlyWarnings);
+  }
+
+  public String getPropertiesFile() {
+    return propertiesFile;
+  }
+
+  public void setPropertiesFile(String propertiesFile) {
+    this.propertiesFile = propertiesFile;
+  }
+
+  /**
+   * Executes the command
+   * @param cmdProcess the command line process
+   */
+  public void execute(CommandLineProcess cmdProcess) {
+    if (getBasePath() == null) {
+      setBasePath(".");
+    }
+    log.debug("initializating dal layer, getting properties from {}", getPropertiesFile());
+    OBPropertiesProvider.getInstance().setProperties(getPropertiesFile());
+
+    final Properties obProperties = OBPropertiesProvider.getInstance().getOpenbravoProperties();
+
+    String pathEtendoRx = obProperties.getProperty("rx.path");
+
+    if (pathEtendoRx == null || pathEtendoRx.isBlank()) {
+      pathEtendoRx = ".";
+    }
+
+    var paths = new GeneratePaths(pathEtendoRx);
+
+    final boolean generateRxCode = Boolean.parseBoolean(obProperties.getProperty("rx.generateCode"));
+    final boolean computedColumns = Boolean.parseBoolean(obProperties.getProperty("rx.computedColumns"));
+    final boolean includeViews = Boolean.parseBoolean(obProperties.getProperty("rx.views"));
+    //
+    log.info("Generate Etendo Rx Code={}", generateRxCode);
```

Comment:
change this to debug

Reply:
This should keep with info level in order to inform the developer running the task